mirgecom/simutil.py

```diff
@@ -434,11 +434,20 @@ def create_parallel_grid(comm, generate_grid):
     return generate_and_distribute_mesh(comm=comm, generate_mesh=generate_grid)
 
 
+def configurate(config_key, config_object=None, default_value=None):
+    """Return a configured item from a configuration object."""
+    if config_object is not None:
+        if config_key in config_object:
+            return config_object[config_key]
+
+    return default_value
+
+
 def compare_files_vtu(
         first_file: str,
         second_file: str,
         file_type: str,
-        tolerance: float = 1e-12
+        tolerance: float
```

Comment:
I think you can keep the default tolerance as 1e-12 now.

Reply:
I thought default tolerance was set in mirgecompare.py? Either as 1e-12 or whatever --tolerance takes in from the command line.

Reply:
Right, but I think there is no harm in providing another default here

Reply:
The way it currently is, the _caller_ must deal with it if the return is  `None`.   In this case, we need to make sure that a default value is passed so that None is _never_ returned.